### PR TITLE
Replace Toblerity/Shapely -> shapely/shapely

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

File: CITATION.txt [hunk 1]
```diff
@@ -6,5 +6,5 @@ below:
   organization = {toblerity.org},
```
Comment (mwtoews) [inline CITATION.txt:6]: What's a better organization name?

Comment (jorisvandenbossche) [reply CITATION.txt:6]: We can maybe also leave out this line completely? (no idea if an "organization" is important for the citation if there is an author already)

Comment (mwtoews) [reply CITATION.txt:6]: yes, leave it out.

File: CITATION.txt [hunk 2]
```diff
@@ -6,5 +6,5 @@ below:
...
   title =     {Shapely: manipulation and analysis of geometric objects},
   year =      {2007--},
-  url = "https://github.com/Toblerity/Shapely"
+  url = "https://github.com/shapely/shapely"
```
Comment (mwtoews) [inline CITATION.txt:9]:
```suggestion
  url =       {https://github.com/shapely/shapely},
```

**Pull request merged**